README.md

```diff
@@ -8,91 +8,31 @@ This repository contains packages needed for a user to
 Project Maintained via  : `turborepo`
 Package manager         : `pnpm`
 
-## Directory Structure
+## Packages
+
+- round-manager: react-frontend for round-manager
```

Comment:
A native speaker might offer more certainty but I think "React frontend" is the usual spelling.

But that aside, calling it a "frontend for round-manager" might make one think that there's a "backend for round-manager" somewhere, whereas round-manager is the app itself.